client/src/components/pages/Settings.tsx

```diff
@@ -89,6 +90,8 @@ const Settings: FC = () => {
 
   const logout = async (): Promise<void> => {
     if (navigation) {
+      if (Platform.OS === 'web') history.go(0);
```

Comment:
@woosanggyu  Would `navigation.reset` method work here?
https://reactnavigation.org/docs/navigation-prop#reset
If this works, I think we can remove the platform checking.

Reply:
I think there is other factor  that `url` is not changed. if `logout` function is called, `signOutAsync()` from `useAuthContext()` is called internally. then remove `token` and reset relayEnvironment. It will trigger re-rendering all components in `AuthProvider`. and `RootStackNavigator` will show a new `View` according to `user` state(In this scenario, `AuthStack`). This flow can not affect `url` in my sight. `RootStack` is just rendering a `Screen`, not a navigation state.

@0916dhkim  This is just my opinion. how do you think?

Reply:
Thank you for your adivce on my pull request. 
I shared your advice with team Cloud to solve the problem.
we conclude it was a problem because `AuthProvider` did not initialize user information. 
So, We are trying to solve the problem through the following actions `setUser(null)` in `signOutAsync` of `AuthProvider`